app/models/coin.rb

```diff
@@ -276,6 +276,24 @@ def sparkline
     end
   end
 
+  def github_stats(force_refresh: false)
+    return nil unless github_repo.present?
```

Comment:
I think you can just do `return` instead of `return nil` to accomplish the same thing.